SparkPost/support-docs
# WE-321 Configure page routing for support docs

**What Changed**
- Adds `index.tsx` and `[...slug].tsx` routes for `docs`
- Adds a redirect from `/` to `/docs`
- Abstracts out layout into a `Layout` component that is shared between docs and momentum
- Abstracts out `ReactMarkdown` component so that is shared between docs and momentum
- Moves all content from `/content/support` to `/content/docs` to reflect existing routing

**How to Verify**
- Run the site locally or visit the deploy preview
- Visit `/` to verify the redirect
- Visit `/docs` to verify the home page route
- Visit `/docs/getting-started/getting-started-sparkpost` to verify single post page content is loading correctly
- Additionally, visit `/momentum` pages to verify the abstractions worked correctly
